### PR TITLE
Fix:Contact page not looking good

### LINKED ISSUE #468: 🐞[bug]: Contact page not looking good
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Contact page is not looking good.
Flex box is not added correctly.

### Expected behavior

Flex box should use correctly

### Add ScreenShots

image
image

### What browsers are you seeing the problem on?

Chrome

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24EXTD contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by gyanshankar1708**
## Description

I had fixed the contact us page, please review it.
Now It is looking better than before.

## Related Issues

- Closes #468 

## Type of PR

- [X] (#468 )

## Checklist

- [X] I have gone through the [contributing guide](https://github.com/Anjaliavv51/Retro)
- [X] I have updated my branch and synced it with project `main` branch before making this PR
- [X] I have performed a self-review of my code
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: Html-files/contact.html
Comment (Anjaliavv51) [inline Html-files/contact.html:1]: y this page has large diff?

Comment (gyanshankar1708) [reply Html-files/contact.html:1]: The document was not formatted and there was everything double so I was facing difficulties. That's why I had used format document option so large difference was showing there....

Comment (Anjaliavv51) [reply Html-files/contact.html:1]: search a way asap , and screen record ur changes from deploy link.

Comment (gyanshankar1708) [reply Html-files/contact.html:1]: @Anjaliavv51 
Here is screen recording 

Before changes

https://github.com/user-attachments/assets/927b4726-5ea4-464c-8849-a97b4b97cc1c

After changes

https://github.com/user-attachments/assets/6a072b40-30c8-4e6b-bb35-f80e0f4217ff

Deployement link - https://deploy-preview-476--rococo-mooncake-0d89f7.netlify.app/

**Comment by Anjaliavv51** [comment]
@gyanshankar1708 it is solved by other person.

**Pull request closed without merging**